app/views/admin/users/index/_filters_modal.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="js-filters-modal hidden">
+    <%= form_with url: admin_users_path, method: :get, local: true, class: "flex flex-col h-100" do |f| %>
+        <div class="flex justify-between items-center border-0 border-b-1 border-base-10 border-solid -mx-7 px-7 pb-2">
+            <h2 class="crayons-subtitle-2">Filters</h2>
+            <button type="button" class="c-btn">Clear filters</button>
+        </div>
+        <div class="py-4">
+            <details class="admin-details">
+                <summary class="py-4 flex justify-between text-uppercase color-base-60 fs-s">Member roles<%= crayons_icon_tag("chevron-down", aria_hidden: true, class: "summary-icon") %></summary>
+                <fieldset class="pb-4">
+                    <legend class="screen-reader-only">Member roles</legend>
+                    <div class="crayons-field crayons-field--radio">
+                      <%= f.radio_button :role, "admin", class: "crayons-radio", role: "radio", id: "admin" %>
```

Comment:
Just an FYI, I think that we're planning on making this a multi-select filter based on [this conversation](https://forem-team.slack.com/archives/C01S4QZUDNC/p1651143585432229?thread_ts=1651134332.643989&cid=C01S4QZUDNC). I've pinged Anuj to ask him to update the design once he's back.